### PR TITLE
DX-2623 Rework Navbar to Use Dropdowns

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

**Comment by ajrice6713** [comment]
Screen Shot 2022-06-03 at 8 55 36 AM
I dont think this is overpass font

**Review by ajrice6713 — DISMISSED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Comment by ajrice6713** [comment]
Hold off on merging until we have stakeholder approval

**Comment by dyeam-bw** [comment]
Menu selection/highlighting issue needs to be fixed. After everything's ready, this can't be merged to `main` yet. It needs to be merged into Dan Miller's branch first so his preview site is fixed for Microsoft and Cisco.

**Pull request merged**
